style.css

```diff
@@ -1 +1,727 @@
+
+/* 
+For the hamburger menu button I have chosen to use the functionality found at https://code-boxx.com/simple-responsive-pure-css-hamburger-menu/
+I Changed IDs to Classes
+*/
+
+/* [ON BIG SCREENS] */
+/* (A) WRAPPER */
+.hamnav {
+    width: 100%;
+    background: #fce0df;
+    font-size:110%; /* Make the hamburger icon bigger */
+ }
+  
+  /* (B) HORIZONTAL MENU ITEMS */
+ .hamitems { display: flex; }
+ .hamitems a {
+    flex-grow: 1;
+    flex-basis: 0;
+    padding: 10px;
+    color: brown;
+    text-decoration: none;
+    text-align: center;
+ }
+ 
+ .hamitems a:hover { background: #401408; }
+  
+  /* (C) HIDE HAMBURGER */
+ .hamnav label, #hamburger { display: none; }
+  
+  /* [ON SMALL SCREENS] */
+    /* (A) BREAK INTO VERTICAL MENU */
+.hamitems a {
+      box-sizing: border-box;
+      display: block;
+      width: 100%;
+      border-top: 1px solid #333;
+}
+  
+    /* (B) SHOW HAMBURGER ICON */
+.hamnav label {
+      display: inline-block;
+      color: brown;
+      background: #fce0df;
+      font-style: normal;
+      font-size: 1.2em;
+      padding: 10px;
+}
+  
+    /* (C) TOGGLE SHOW/HIDE MENU */
+.hamitems { display: none; }
+.hamnav input:checked ~ .hamitems { display: block; }
+
+.menugap {
+    padding-right:16em;
+}  
+
+/**** CSS CODE FOR HAMBURGER MENU ENDS ****/
+
+.menu-quote-container {
+    padding-bottom:1em;
+}
+
+
+
+.FooterContainer {
+    background-color: #1f42c3;
+    grid-area: footer;
+    max-width: 480px;
+}
+
+  
+/*
+Buttons used for the Footer
+*/
+
+.ButtonsContainer {
+    margin:1em 0 1em 1em;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap:1rem;
+}
+
+.footer-button {
+    width:26%;
+    border: 2px solid black;
+    border-radius: 5px;
+    background-color: #95d4e7;
+    color: black;
+    padding: 0px 0px;
+    font-size: 16px;
+    cursor: pointer;
+    text-align: center;
+  }
+
+.footer-button:hover {
+    background: skyblue;
+}
+
+  .copyright {
+      padding-top:1rem;
+      margin-bottom:1rem;
+      font-size:14px;
+      color: mistyrose;
+      text-align:center;
+  }
+
+
+/* THE GRIDS AND MEDIA QUERIES START HERE */
+
+.homepage-container {
+    width: 320px;
+    grid-template-columns: repeat(4, 1fr);
+    /* grid-template-rows: 360px 180px 400px 470px 150px; */
+    grid-template-rows: auto;
+    display: grid;
+    grid-template-areas:
+    "hero       hero        hero        hero"
+    "content    content     content     content"
+    "cake      cake         cake        cake"
+    "flex       flex        flex        flex"  
+    "footer     footer      footer      footer";
+} 
+
+
+
+
+/* TABLET: */
+@media screen and (min-width: 481px) {
+
+   /*  
+    We only define rules which are DIFFERENT from the mobile version defined above. 
+  */   
+
+          .homepage-container {
+            margin:3em;
+            width:100%; /* Use the full screen */
+            display: grid;
+            grid-template-columns: repeat(4, 1fr);
```

Comment:
You don't need to repeat the rules that haven't changed from the mobile version. Only over-write those which have changed.